src/pages/Home/HomePage.tsx

```diff
@@ -57,9 +62,22 @@ const HomePage = () => {
     }
   };
 
-  const handleChangeSavedLocation = useCallback((value:string) => {
-    setSavedLocation(value);
-  },[setSavedLocation]);
+  const handleChangeSavedLocation = useCallback(
+    (value: string) => {
+      setChosenCity(value);
+      setCurrentLocation(value);
+    },
+    [setCurrentLocation],
+  );
+
+  const updateCoordinates = useCallback(async () => {
+    const { data } = await refetch();
+    data && setCurrentCoordinates(`[${data.location.lat}, ${data.location.lon}]`);
```

Comment:
**chore** this function is redundant

current coordinates are additional data and it can be obtained when data is loaded (which was done before)

Reply:
Yeah, you're right